index.html

```diff
@@ -0,0 +1,336 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Glorious Vineyard Of Christ Chapel</title>
+    <link rel="stylesheet" href="assets/css/desk-home.css" />
+    <link rel="stylesheet" href="assets/css/mob-home.css" />
+
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'/>
+  </head>
+  <body>
+    <header>
+      <ul class="header-above">
+        <a>
+          <li class="header-above-content">
+            <img src="assets/images/icons/facebook.png" />
+          </li>
+        </a>
+        <a>
+          <li class="header-above-content">
+            <img src="assets/images/icons/twitter.png" />
+          </li>
+        </a>
+        <li class="header-above-content">English</li>
+        <li class="header-above-content">My Page</li>
+        <li class="header-above-content">Logout</li>
+      </ul>
+      <nav class="header-below">
+        <nav class="mob-header">
+          <img
+            src="assets/images/icons/mobile_menu 1.png"
+            class="hamburger"
+            id="nav-button"
+            width="40"
+            height="40"
+          />
+          <a class="links links-hamburger" href="#home">
+            <div class="logo">
+              <img src="assets/images/icons/Logo.jpg" width="200" height="70" />
+            </div>
+          </a>
+        </nav>
+        <ul class="header-below-list mobile">
+          <li class="header-below-content close">
+            <span class="material-symbols-outlined"> close </span>
+          </li>
+          <a href="about.html" class="links">
+            <li class="header-below-content">About</li>
+          </a>
+          <a href="#program" class="links">
+            <li class="header-below-content">Program</li>
+          </a>
+          <li class="header-below-content">Join</li>
+          <li class="header-below-content">Sponsor</li>
+          <li class="header-below-content">News</li>
+          <li class="header-below-content cc-btn">CC Campaign</li>
+        </ul>
+      </nav>
+    </header>
+    <section class="conference-page">
+      <div class="background-img"></div>
+      <div class="conference-content">
+        <h1 class="presents">GVCC Presents......</h1>
+        <h1 class="conference-intro">
+          GLOBAL PROPHETIC <br />CONVOCATION <br />2024
+        </h1>
```

Comment:
- [ ] For semantic purposes, please ensure that you use only one `h1` in your entire HTML file. :slightly_smiling_face: